suite/tests/CMakeLists.txt

```diff
@@ -2233,6 +2233,9 @@ if (CLIENT_INTERFACE)
       # i#2119: test invoking the app's handler on a DR fault.
       tobuild_api(api.static_crash api/static_crash.c "-unsafe_crash_process" "" OFF ON)
       target_link_libraries(api.static_crash ${libmath})
+      # i#2346: sideline therad on Windows
```

Comment:
I don't understand this comment: it doesn't describe this test.  I think you mean it to explain why this is inside NOT WIN32 but it needs to explain that and use XXX or sthg.